lib/CompaniesHouseGenericService.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2017, Will Evans.
+ * Licensed under the MIT License.
+ */
+const axios = require('axios');
+
+class CompaniesHouseGenericService {
+
+    constructor(apiKey){
+        this.apiKey = apiKey;
+    }
+
+    /**
+     * This method calls the companies house API to return data for a specific link e.g one returned in Company Profile
+     * @param link
+     * @return {Promise}
+     */
+    getLink(link){
+        return new Promise(
+            (resolve, reject) => {
+                axios({
+                    method:'get',
+                    url: `https://api.companieshouse.gov.uk${link}`,
```

Comment:
Quick question on this line, what format is the `link` variable? Am I right in assuming everything that comes into it will append correctly at the end of the URL? Or should there be a trailing slash at the end of `.gov.uk`?

Thanks for the PR by the way :)